### DISCUSSION TITLE
Is the mintpy output orthometric?

### DISCUSSION THREAD

**Discussion by Rejoicet** [insarlab/MintPy#1123] [Q&A]
When imagery is collected via a satellite or aerial sensor, the location information (such as height) is typically measured above an ellipsoidal model of the earth, whereas many available digital elevation models (DEM) are modeled using an orthometric model. Mintpy (SBAS) uses satellite imagery as well as DEMs to produce the output. So finally, is the output measured using the ellipsoidal model or an orthometric model? Does mintpy do any conversions internally to convert either satellite measurements to orthometric or DEM to ellipsoidal before producing the output?

I am using the mintpy with ASF hyp3 product

**Comment by yunjunz** [comment]
MintPy does not do any conversions internally to the InSAR products coordinate system. 

What you said is common in optical remote sensing. In radar remote sensing, or in ASF hyp3 product as I know, the InSAR products is produced with respect to the DEM already.
